__mocks__/kf-api-release-coordinator/mocks.js

```diff
@@ -0,0 +1,30 @@
+import {GET_STUDY_RELEASES, GET_RELEASED_STUDY} from '../../src/state/queries';
+import getStudyReleases from './responses/getStudyReleases.json';
+import allReleaseStudies from './responses/allReleaseStudies.json';
+
+export const coordMocks = [
```

Comment:
Can we avoid this pattern? It's extremely difficult to tell what queries are being mocked from the tests themselves.